language-design.md

```diff
@@ -69,8 +157,20 @@ The Yang and Rabkin article talks mainly about general-purpose languages. In
 what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
-
-
+DSLs develop their own hierarchy because of the wide range of uses for DSLs, in
+some ways even more pronounced than the hierarchy in general-purpose languages
+discussed in Yang and Rabkin. Some DSLs are made for people who have limited or
+no programming experience. It's easy to stereotype these languages as easier and
+lesser programming languages than some more intense DSLs that are difficult to learn 
+without extensive programming experience. This develops a hierarchy amongst DSLs, 
+where the languages that require more programming knowledge are considered objectively 
+better than those that are made for artists, web developers, and other less technical users.
+Instead of choosing the best language for a project, programmers may be attached
+to a DSL and use a suboptimal language because of pride instead of the project.
+
+Moreover, if DSLs are internal, they carry the stereotypes of their host language. 
+For instance, if people think Python is "for noobs" then the internal DSL in Python
+would also be "for noobs" [Yang and Rabkin, 2015].
```

Comment:
Ideally, if your internal DSL shakes the host language's flavor enough, then I think it'll get away from the preconceptions and stereotypes against the host language.